src/ScatterTab.tsx

```diff
@@ -7,7 +7,7 @@ import InvertColorsIcon from "@material-ui/icons/InvertColors";
 import InvertColorsOffIcon from "@material-ui/icons/InvertColorsOff";
 import SaveIcon from "@material-ui/icons/Save";
 import CloudUploadIcon from "@material-ui/icons/CloudUpload";
```

Comment:
Not used anymore- Shouldn't be imported.

Reply:
```suggestion

```